core/rpc_client.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/env python
```

Comment:
Shouldn't need this unless this file is meant to be executed directly from the command line.

`rpc_server.py` and both unit test files also shouldn't need this, I think.

Only `control.py` does need it, otherwise strange things happen when you try to run it directly.